test/functionalTest/cases/1080_batch_query_context/batch_query_context_scopes.test

```diff
@@ -146,8 +144,8 @@ echo
 echo
 
 
-echo "08. POST /v2/op/query, with E.* with scopes q:!type and q=A5==45, see E5/none"
```

Comment:
An old leftover? Not sure, but I think that `!type` comes from times where enity type optionality was possible, but now that is not possible (given that although the user creates an entity without type CB will set a default type for it).

Is my reasoning correct?